src/Benchmarks/Benchmarks.xproj

```diff
@@ -9,7 +9,7 @@
     <ProjectGuid>ec62acf4-8b19-41c2-b699-d75cab7763df</ProjectGuid>
     <RootNamespace>Benchmarks</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\</OutputPath>
   </PropertyGroup>
```

Comment:
VS wanted to make this change, so I figured why not. This output path was generated as part of an RC1 xproj, and won't work with dotnet CLI + p2p references which is why VS changed it.